config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -191,6 +193,7 @@ spec:
               type:
                 type: string
             required:
+            - ProvisioningOperationId
```

Comment:
Please change the json case to use camelCase and re-run `make generate//manifests` targets to update these values.